doc/pages/expansions.asciidoc

```diff
@@ -200,9 +200,14 @@ The following expansions are supported (with required context _in italics_):
     quoted list of the names of currently-open buffers (as seen in
     `%val{bufname}`)
 
-*%val{bufname}*::
+*%val{buffile}*::
+    _in buffer, window scope_ +
+    full path of the file or same as `%val{bufname}` when there’s no
+    associated file
```

Comment:
I assume you didn't intend to replace the documentation for `bufname` with a second copy of the `buffile` description?

Reply:
hehe nope, will fix right away

Reply:
fixed now, thanks